docs/topics/funding.md

```diff
@@ -115,36 +111,50 @@ REST framework continues to be open-source and permissively licensed, but we fir
 
 ---
 
-## Making the business case
+## What funding has enabled so far
 
-Our [successful Kickstarter campaign](https://www.kickstarter.com/projects/tomchristie/django-rest-framework-3) demonstrates the impressive cost-reward ratio of shared funding of open-source software.
+* The [3.4](http://www.django-rest-framework.org/topics/3.4-announcement/) and [3.5](http://www.django-rest-framework.org/topics/3.5-announcement/) releases, including schema generation for both Swagger and RAML, a Python client library, a Command Line client, and addressing of a large number of outstanding issues.
+* The [3.6](http://www.django-rest-framework.org/topics/3.6-announcement/) release, including  JavaScript client library, and API documentation, complete with auto-generated code samples.
+* Tom Christie, the creator of Django REST framework, working on the project full-time.
+* Over 320 closed issues on [GitHub](https://github.com/tomchristie/django-rest-framework) since Tom Christie started working on the project full-time (June 2016 - March 2017).
+* A community & operations manager position part-time for 4 months, helping mature the business and grow sponsorship.
+* Contracting development time for the work on the JavaScript client library and API documentation tooling.
 
-With *typical corporate fundings of just £100-£1000 per organization* we successfully delivered:
+---
 
-* The comprehensive serializer redesign, and **version 3.0 release**.
-* Ongoing triage and community support, **closing over 1600 tickets**.
-* Substantial improvements to the Browsable API.
-* The admin interface.
-* A new pagination API including offset/limit and cursor pagination implementations, plus on-page controls.
-* A versioning API, including URL-based and header-based versioning schemes.
-* Support for customizable exception handling.
-* Support for Django's PostgreSQL HStoreField, ArrayField and JSONField.
-* Templated HTML form support, including HTML forms with nested list and objects.
-* Internationalization support for API responses, currently with 27 languages.
-* The metadata APIs for handling `OPTIONS` requests and schema endpoints.
-* Numerous minor improvements and better quality throughout the codebase.
+## What future funding will enable
 
-This incredible level of return on investment is *only possible through collaboratively funded models*, which is why we believe that supporting our paid plans is in everyone's best interest.
+* Realtime API support, using WebSockets. This will consist of documentation and support for using REST framework together with Django Channels, plus integrating WebSocket support into the client libraries.
+* Better authentication defaults, possibly bringing JWT in to the core package.
```

Comment:
I think we could update this to "...possibly bringing JWT & CORs support into..."